src/pages/PaymentPage/PaymentPage.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from 'react'
+
+import { AddEditCraditCard } from 'pages/AddEditCraditCard'
+import { PaymentMain } from 'pages/PaymentMain'
+import { Loading } from 'components/atoms/Loading'
+import { ErrorIcon } from 'components/atoms/ErrorIcon'
+import useCards from 'hooks/useCards'
+import { CardType } from 'models/card.model'
+import { INIT_CARD_VALUE } from 'constants/card'
+
+const PaymentPage = () => {
+  const [isAddEditCard, setIsAddEditCard] = useState(false)
+  const { loading, error, cards, addCard } = useCards()
+  const [selectCard, setSelectCard] = useState(INIT_CARD_VALUE)
+
+  const toggleIsAddCard = (isAdd: boolean) => {
+    setIsAddEditCard(isAdd)
+  }
+
+  if (loading) {
+    return <Loading />
+  }
+
+  if (error) {
+    return <ErrorIcon message={error.message} />
+  }
```

Comment:
```suggestion
  if (loading) {
    return <Loading />
  }

  if (error) {
    return <ErrorIcon message={error.message} />
  }
  
  const toggleIsAddCard = (isAdd: boolean) => {
    setIsAddEditCard(isAdd)
  }

```

함수 선언을 더 나중에 해도 되겠네요.
loading과 error로 early return이 일어나지 않고 그 뒤에 함수를 선언해도 괜찮을 것 같아요

그리고 `toggleIsAddCard` 함수 이름이 조금 와닿지 않는 것 같아요.
토글이라는 건 한 번 일어날 때 true, false를 왔다갔다 하는 느낌인데 
지금 함수는 파라미터를 받고 그 값으로 변경하는 친구니까 toggle보다는 change 혹은 set이 조금 더 의미적으로 맞겠네요

저는 이럴 때 그냥

```ts
const editStart = () => setIsAddEditCard(true);
const editEnd = () => setIsAddEditCard(false);
```

이런식으로 조금 더 명시적으로 알아보기 쉽게 함수를 두 개 작성을 하는 것 같아요.